core/src/com/geekbrains/td/GameScreen.java

```diff
@@ -61,25 +74,34 @@ public InfoEmitter getInfoEmitter() {
         return infoEmitter;
     }
 
+    public Antiking getAntiking() {
+        return antiking;
+    }
+
     public King getKing() {
         return king;
     }
 
     @Override
     public void show() {
         this.player = new Player();
-        this.king = new King();
         this.mousePosition = new Vector2(0, 0);
         this.particleEmitter = new ParticleEmitter();
         this.font18 = Assets.getInstance().getAssetManager().get("fonts/zorque18.ttf");
         this.font24 = Assets.getInstance().getAssetManager().get("fonts/zorque24.ttf");
+        this.font72 = Assets.getInstance().getAssetManager().get("fonts/zorque72.ttf");
+        this.king = new King();
+        this.antiking = new Antiking(this);
         this.bulletEmitter = new BulletEmitter(this);
         this.map = new Map("level01.map");
         this.monsterEmitter = new MonsterEmitter(this);
         this.turretEmitter = new TurretEmitter(this);
         this.infoEmitter = new InfoEmitter(this);
         this.selectedCellTexture = Assets.getInstance().getAtlas().findRegion("cursor");
         this.createGUI();
+        level = 1;
+        isGameOver = false;
```

Comment:
Boolean field is `false` by default. Golden rule: do not add any code until you should to.